### DISCUSSION TITLE
How to show red spheres around the agent when there are violations in rendered videos?

### DISCUSSION THREAD

**Discussion by hoangclo** [PKU-Alignment/safety-gymnasium#67] [Q&A]
In safety-gymnasium, when I render the videos of the agent, I don't see red spheres around the agent when there are violations with boundaries or blocks (safety gym does). 

Anyone knows how I can show red spheres like safety-gym does?

**Comment by muchvo** [comment]
Hi, can you provide more details about your version of safety-gymnasium and your system environment?

**Reply by hoangclo** [reply]
I am using ver 0.4.1 in Window 10. I can't see the red spheres in all environments (e.g. Goal, Button, Circle).

I attached an example of the issue:

![image](https://github.com/PKU-Alignment/safety-gymnasium/assets/127832820/4479c15c-366a-445d-9ef9-d1314de3c99a)

**Reply by muchvo** [reply]
You could try to update to v1.0.0. This bug has been fixed.

**Reply by hoangclo** [reply]
Okay. Thank you for the information
